arch/arm/src/stm32wl5/stm32wl5_flash.c

```diff
@@ -359,6 +343,31 @@ ssize_t up_progmem_eraseblock(size_t block)
     }
 }
 
+ssize_t up_progmem_ispageerased(size_t page)
```

Comment:
why change the order of up_progmem_ispageerased and up_progmem_eraseblock? it make the change is hard to compare.

Reply:
Orginally stm32wl5_flash.c was based on code copied from stm32l5, but it turned out stm32l4 is more compatible with stm32wl5, so I copied over file from l4 and then adapted the code to work with wl5. I think we should consider files in arch/* as new files, not diffs.

Reply:
OK